package.json

```diff
@@ -24,6 +24,7 @@
   },
   "devDependencies": {
     "@babel/eslint-parser": "^7.18.2",
+    "@babel/plugin-transform-class-properties": "^7.22.5",
```

Comment:
These changes belong in a different PR. Also note that instead of maintaining these deps here and syncing them with a custom `babel.config.js`, a simpler/safe option is to just leverage the config used by stripes-webpack and exported via stripes-cli, [e.g. in stripes-core](https://github.com/folio-org/stripes-core/blob/a5e888b9a102d25f471dc5306feba02cc2e2e82b/test/jest/babel.config.js):
```
const { babelOptions } = require('@folio/stripes-cli');

// ... customizations

module.exports = {
  ...babelOptions,
};
```